sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -780,11 +794,8 @@ class DataSourceV2SQLSuite
   test("ShowNamespaces: default v2 catalog is not set") {
     spark.sql("CREATE TABLE testcat.ns.table (id bigint) USING foo")
 
-    val exception = intercept[AnalysisException] {
-      sql("SHOW NAMESPACES")
-    }
-
-    assert(exception.getMessage.contains("No default v2 catalog is set"))
+    // The current catalog is resolved to a v2 session catalog.
+    testShowNamespaces("SHOW NAMESPACES", Seq())
```

Comment:
Note that the result of this is `Seq()`, and not `Seq("default")` due to the following: 

https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2SessionCatalog.scala#L190-L192

Note that `listNamespaces(Array("default"))` is called since the current namespace is "default".

Reply:
wait, I think the previous conclusion was to ignore current name space and always list the root namespaces for `SHOW NAMESPACES`. cc @rdblue

Reply:
@cloud-fan you are right. I am fixing this. Thanks for catching this.